components/about_build_modal/about_build_modal.tsx

```diff
@@ -102,9 +102,9 @@ export default class AboutBuildModal extends React.PureComponent<Props, State> {
                 <a
```

Comment:
Line 100 above says `defaultMessage='Join the Mattermost community at '`, shouldn't we be changing that as well?

Reply:
Oh I tried to not change any of the default messages, I don't think i did? There was a lot of text I had to change that wasn't a `FormattedMessage` and was just plain text, but anything with a `defaultMessage` property I tried to leave as is and only change the i18n file